### PR TITLE
Updated aws-marketplace.md with links

### PR DESCRIPTION
**Pull request by kevinayres**
Updated aws-marketplace.md description and links to include GSG and Demo Video

### CODE REVIEW

File: docs/getting-started/quick-start-guides/deploy-rancher-manager/aws-marketplace.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ title: Rancher AWS Marketplace Quick Start
 description: Use Amazon EKS to deploy Rancher server.
 ---
 
-There is now an additional way for you to deploy the Rancher server in AWS by using Amazon EKS. To learn more, see our [Amazon Marketplace listing](https://aws.amazon.com/marketplace/pp/prodview-go7ent7goo5ae).
+You can now quickly deploy Rancher server in AWS by using Amazon EKS. To learn more, see our [Amazon Marketplace listing](https://aws.amazon.com/marketplace/pp/prodview-go7ent7goo5ae) or review the [Getting Started Guide] (https://documentation.suse.com/trd/kubernetes/single-html/gs_rancher_aws-marketplace/) in SUSE official docs, or watch the [Demo Video] (https://youtu.be/9dznJ7Ons0M) to be walked through the process.
```
Comment (martyav) [inline docs/getting-started/quick-start-guides/deploy-rancher-manager/aws-marketplace.md:6]:
```suggestion
Amazon Elastic Kubernetes Service (EKS) can quickly deploy Rancher server to Amazon Web Services (AWS). To learn more, see our [Amazon Marketplace listing](https://aws.amazon.com/marketplace/pp/prodview-go7ent7goo5ae). Instructions are available at [Deploy Rancher on Elastic Kubernetes Service (EKS)] (https://documentation.suse.com/trd/kubernetes/single-html/gs_rancher_aws-marketplace/). See a walkthrough of the process by watching the [demo] (https://youtu.be/9dznJ7Ons0M). 
```

**Review by martyav — APPROVED** [review]
I just made some changes to the proposed content. Looks good.

**Comment by btat** [comment]
In the docs page linked I see `Rancher Setup is designed to deliver the current stable version of Rancher. At the time of this writing, Rancher 2.6.10 is deployed`. Given that's the case, it might make more sense to update the 2.6 version of the marketplace page in our docs and remove the marketplace page in 2.7 until the tool starts installing that. Thoughts @martyav?

@kevinayres do you happen to know when the Rancher Setup tool will start installing the 2.7.x line or whether there are plans to allow the Rancher version to be configurable?

**Comment by martyav** [comment]
@btat I think that's a reasonable change, we just need to keep an eye on when the tool updates.

**Pull request merged**